backend/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM tiangolo/uvicorn-gunicorn-starlette:python3.7
```

Comment:
Two notes: 

Python 3.8 was released not so long ago so you might want to consider using it (it has a few nice features). Unfortunately there is no image for this version updated by this maintainer so you can use a simple configuration (you don't need to worry about performance so `gunicorn` step can be skipped and you can write your own simple `Dockerfile` based on 3.8)

Consider using a variant of this image based on alpine: `:python3.7-alpine3.8`. It weighs a lot less and you probably don't need the features from the current image.

Reply:
There is already a PR submitted to update the image to 3.8 so be on the lookout when it gets merged :) https://github.com/tiangolo/uvicorn-gunicorn-docker/pull/19

Reply:
I'll keep my eye on uvicorn-gunicorn-docker, but I will just use simple Dockerfile (based on the one provided by tiangolo) to support python3.8-alpine and pipenv environment.